libs/docs/core/grid-list/examples/combo-select/grid-list-combo-select-example.component.html

```diff
@@ -0,0 +1,241 @@
+        <button fd-button fdkFocusableItem label="SingleSelectRight" value="SingleSelectRight"></button>
+        <button fd-button fdkFocusableItem label="Delete" value="Delete"></button>
+    </fd-segmented-button>
+    <br />
+    <br />
+    @switch (selectionMode) {
+        @case ('None') {
+            <fd-grid-list>
+                <fd-grid-list-title-bar [title]="'Products'">
+                    <span fd-grid-list-title-bar-additional-title-item>{{ '(' + list.length + ')' }}</span>
+                    <fd-grid-list-title-bar-spacer></fd-grid-list-title-bar-spacer>
+                    <button
+                        fd-button
+                        fdType="transparent"
+                        glyph="sort"
+                        aria-labelledby="grid-list-title-bar-1 grid-list-button-1"
+                        aria-label="Sort"
+                        title="Sort"
+                        fdCompact
+                        (click)="showAlert('Sort')"
+                    ></button>
+                    <button
+                        fd-button
+                        fdType="transparent"
+                        glyph="filter"
+                        aria-label="Filter"
+                        title="Filter"
+                        fdCompact
+                        (click)="showAlert('Filter')"
+                    ></button>
+                    <button
+                        fd-button
+                        fdType="transparent"
+                        glyph="group-2"
+                        aria-label="Group"
+                        title="Group"
+                        fdCompact
+                        (click)="showAlert('Group')"
+                    ></button>
+                </fd-grid-list-title-bar>
+                @for (item of list; track item) {
+                    <fd-grid-list-item
+                        [type]="item.type"
+                        [counter]="item.counter"
+                        [title]="item.title"
+                        [description]="item.description"
+                        (navigate)="navigate()"
+                    >
+                        <fd-avatar
+                            fd-grid-list-item-image
+                            image="https://picsum.photos/id/1062/300/200"
+                            size="s"
+                        ></fd-avatar>
+                        <ng-template fd-grid-list-item-body>
+                            <p>781 Main Street</p>
+                            <p>Anytown, SD 57401</p>
+                            <p>USA</p>
+                            <a [routerLink]="['./']" fd-link tabindex="0">john_li&#64;example.com</a>
+                        </ng-template>
+                    </fd-grid-list-item>
+                }
+            </fd-grid-list>
+            <span aria-hidden="true" [style.display]="'none'" id="grid-list-button-1">Sort</span>
```

Comment:
No fault of your PR, but I'm not sure why we had this set up with these hidden spans, can just use data binding on the button's aria-label